beetsplug/importreplace.py

```diff
@@ -68,9 +68,11 @@ def _albuminfo_received(self, info: AlbumInfo):
         for track in info.tracks:
             self._trackinfo_received(track)
 
-    def _replace_field(self, text: str, replacements: [(Pattern, str)]) -> str:
-
-        return reduce(self._replace, replacements, text)
+    def _replace_field(self, text: str|list, replacements: [(Pattern, str)]) -> str|list:
```

Comment:
Is this syntax compatible with <3.10? Currently the `setup.py` for this plugin indicates it supports beets >=1.4.7, and I think that version supports Python 3.6. So I'd like to keep it compatible with older versions as well.

Reply:
It is not; the `X|Y` shorthand for `Union[X, Y]` was added in 3.10 with [PEP 604](https://peps.python.org/pep-0604/)